options.go

```diff
@@ -67,8 +67,7 @@ func WithConverter(fieldName string, fn ValidateConvertFunc) Option {
 	return func(o *options) error {
 		switch {
 		case fieldName != "" && !isNil(fn):
-			o.withValidateConvertFn = fn
-			o.withValidateConvertColumn = fieldName
+			o.withValidateConvertFns[fieldName] = fn
```

Comment:
should we prevent duplicate converters?  for example, if someone specifies two
converters for the `name` field?

Reply:
Added duplicate detection, if the user specifies a converter for the same field multiple times, an error will be returned